### ISSUE TITLE
Debug builds should not auto-run the installer project.

### ISSUE THREAD

**Issue by mslagle** [oleg-shilo/wixsharp#417]
When debugging an installer project, it is impossible to step through due to the post-build target that auto runs the installers.  I was trying to step through the installer project but it kept failing because my path to binaries was wrong.  I either needed to change your targets file or create console line output to debug my path issues.

**Comment by oleg-shilo** [comment]
Correct. The VS project primary objective is to build but not debug. Thus if debugging is required you need to use `Debug.Assert` or change the "WixSharp" package to "WixSharp.bin", which has no "*.target". Basically it is an ordinary Console app.

**Comment by mslagle** [comment]
Change the title.  Basically, if I use wixsharp.bin as my package, the executable runs the bin directory instead of the project directory, causing all my paths to harvest files, images, etc to be wrong and need modified.  Thus, I need to modify the wixsharp.bin targets file to set the startup path (and maybe add startup arguments) so debug builds match release builds.

**Comment by oleg-shilo** [comment]
> Basically, if I use wixsharp.bin as my package, the executable runs the bin directory...

If you use `WixSharp` package the working and output directory for your post-build run is set for you by the `*.targets`. The same as for and application executed in VS as post-puild.

Though if you are not using _targets_ (`WixSharp.bin`) then you need to do it by yourself. either in project settings dialog (Debug > Working Directory = ..\..) or even better to do it in the code:
```
project.OutDir
project.SourceBaseDir
```
